api/src/main/java/io/opentelemetry/OpenTelemetry.java

```diff
@@ -56,7 +58,18 @@
    * @since 0.1.0
    */
   public static Tracer getTracer() {
-    return getInstance().tracer;
+    return getTracer("defaultInstrumentationName", "defaultInstrumentationVersion");
```

Comment:
Should we really keep the parameterless `getTracer()`? I'm afraid this will result in many implementers just taking the quick and easy `getTracer()` which pops up first in their IDE without considering to assign a proper name.

Reply:
I'm really happy either way. I'm not sure how to implement this part of the OTEP, in either case:
```
If no name (null or empty string) is specified, following the suggestions in "error handling proposal", a "smart default" will be applied and a default Tracer / Meter implementation is returned.
```

Reply:
Since I removed these methods, there are now no default instrumentation names in the codebase.

Reply:
Thanks! I'd prefer to not make the fallback more convenient to use than the suggested way - now one has to intentionally pass `null` which should be fine.